modules/op-about-triggers.adoc

```diff
@@ -84,14 +84,35 @@ spec:
           claimName: source-pvc
 ----
 
-<1> TriggerTemplate API version `v1alpha1`.
+<1> Trigger template API version `v1alpha1`.
 <2> Specifies the type of Kubernetes object. In this example, `TriggerTemplate`.
-<3> Unique name to identify this TriggerTemplate.
-<4> Parameters supplied by the TriggerBinding or EventListerner.
-<5> List of Resource templates created for the Pipeline from the parameters received in the TriggerBinding or EventListener.
+<3> Unique name to identify the `TriggerTemplate` resource.
+<4> Parameters supplied by the `TriggerBinding` or `EventListerner` resources.
+<5> List of resource templates created for the pipeline from the parameters received through the `TriggerBinding` or `EventListener` resources.
```

Comment:
The same as above... we don't create templates, we provide templates that are used for creating resources, this sentence is confusing for me